Doc/reference/datamodel.rst

```diff
@@ -1578,8 +1578,8 @@ Class Binding
    ``A.__dict__['x'].__get__(None, A)``.
 
 Super Binding
-   If ``a`` is an instance of :class:`super`, then the binding ``super(B,
-   obj).m()`` searches ``obj.__class__.__mro__`` for the base class ``A``
+   If ``a`` is an instance of :class:`super`, then the binding ``super().m()``
```

Comment:
This should keep the explicit 2-arg form, and then add a follow-up sentence reminding readers how that relates to the zero-arg form (i.e. it's implicitly `super(__class__, <first parameter>)` when inside a class definition).

However, to make it clearer what's going on, it's probably worth breaking it up as ``a = super(B, obj); a.x()``, rather than assuming the reader will automatically realise that the `super()` call is defining ``a`` and that ``m`` here is equivalent to ``x`` in the preceding examples.